clustermesh/clustermesh.go

```diff
@@ -2082,6 +2082,97 @@ func (k *K8sClusterMesh) ConnectWithHelm(ctx context.Context) error {
 	return nil
 }
 
+func (k *K8sClusterMesh) DisconnectWithHelm(ctx context.Context) error {
```

Comment:
Refactoring the duplication here with `ConnectWithHelm` might be in order, as it would have prevented the `Connect` bug below.

Reply:
This code being so sequential is a bit hard to refact. Did a quick pass. Let's see if we're happier.